src/desktop/components/auth_modal/test/view.coffee

```diff
@@ -138,5 +140,25 @@ describe 'AuthModalView', ->
       @view.onSubmitSuccess @view.user, { success: 200 }
       @submitSpy.should.be.calledOnce
 
+  describe 'GDPR compliance', ->
+    beforeEach ->
+      @view.state = new Backbone.Model mode: 'register'
+      @view.user = new LoggedOutUser
+      @view.$el.html $ "<form>" + render('register')(
+        copy: new Backbone.Model
+        sd: CSRF_TOKEN: 'csrfoo', AP: loginPagePath: 'foo'
+      ) + "</form>"
+      # @view.renderInner()
+
+
 
+    xit 'requires user accept terms for registration via email', ->
```

Comment:
@mzikherman - We were unable to write tests for this TOS form validation. Any advice here? The plan is to set up a new component, probably in reaction after this AB test, so we could focus on them there.

Reply:
No, this seems fine to me, setting the input values looks good and then, you should be able to do a submit/validation, or whatever. What's the error message?

Reply:
Basically the HTML5 `Form.checkValidity()` api and native browser handling of validation messages doesn't seem to be available in mocha tests.

Reply:
@mzikherman right I know benv uses the headless browser and we were wondering if those browser validatations are exposed in that environment.

Reply:
@erikdstock You mentioned that the HTML5 form validation API doesn't allow us to style the popup that displays the error. Is there a way to suppress those and show our custom error box instead? Wouldn't want us to commit to this solution and find out we're stuck with their default UI afterwards.

Reply:
There's a `:invalid` pseudo-class that lets style the `input` elements not the popup from what I've seen
https://developer.mozilla.org/en-US/docs/Web/CSS/:invalid